tests/VirtoCommerce.CatalogModule.Tests/CatalogAuthorizationHandlerTests.cs

```diff
@@ -78,15 +80,214 @@ public async Task Handle_Permission_With_Incorrect_Scope_Catalog()
             var dynamicAssociation = new DynamicAssociation();
 
             var context = CreateAuthorizationHandlerContext(
-                "test:permission",
-                "test:permission|[{\"catalogId\":\"testCatalog\",\"type\":\"SelectedCatalogScope\",\"label\":\"Electronics\",\"scope\":\"testCatalog\"}]",
+                _permission,
+                $"{_permission}|[{{\"catalogId\":\"testCatalog\",\"type\":\"SelectedCatalogScope\",\"label\":\"Electronics\",\"scope\":\"testCatalog\"}}]",
                 dynamicAssociation);
 
             await catalogAuthorizationHandler.HandleAsync(context);
 
             Assert.False(context.HasSucceeded);
         }
 
+        [Fact]
+        public async Task Handle_Permission_Without_Scope()
```

Comment:
Please rename all the test methods according to the MS recommended convention:
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests
In our case expected result is missing in the name - it could be 
`Handle_PermissionWithoutScope_Succeeded`.